cocos/ui/UIPageView.cpp

```diff
@@ -59,6 +59,19 @@ PageView* PageView::create()
     CC_SAFE_DELETE(widget);
     return nullptr;
 }
+
+PageView* PageView::create(bool adjustChildSize)
```

Comment:
why pageview need the argument `adjustChildSize`?
We only need to call the `setAdjustChildSizeEnabled` in ListView?